### PR TITLE
Updating localized strings and package*.json with latest changes in main

### PR DESCRIPTION
**Pull request by paninaro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -168,5 +168,5 @@
             "last 1 safari version"
         ]
     },
-    "version": "1.2.6-dev67"
-}
+    "version": "1.2.11-dev20"
```
Comment (paninaro) [inline package.json:171]: I'm not sure where this change comes from or if we want to take it. Looks like it also removed the trailing newline at the end of the file.

File: src/locales/be-BY/messages.po [hunk 2]
```diff
@@ -3050,10 +3050,6 @@ msgstr "сінхр."
 msgid "syncing"
 msgstr "ідзе сінхр."
 
-#: src/components/plot/add/PlotAddChoosePlotter.tsx:89
-msgid "thirdparty.plotter.disclaimer"
-msgstr ""
-
```
Comment (paninaro) [inline src/locales/be-BY/messages.po:3056]: This string was inadvertently added when `chiaplotters_gui` was merged. I did a batch find/replace to remove them and then re-ran `npm run locale:extract && npm run locale:compile`

File: package.json (hunk 1 above)
Comment (emlowe) [reply package.json:171]: it's best to remove the version string from package.json.

Comment (paninaro) [reply package.json]: Restored

**Review by emlowe — APPROVED** [review]

**Pull request closed without merging**